swagger.yaml

```diff
@@ -49,6 +74,22 @@ paths:
           description: "Recipe not found"
         500:
           $ref: '#/responses/InternalError'
+    put:
+      summary: "Update a recipe"
+      description: |
+        Update a recipe by getting the specific recipe by id and update in mongo
+      parameters:
+      - $ref: '#/parameters/recipe_id' 
+      - $ref: '#/parameters/new_recipe'
```

Comment:
Same as above

Reply:
Resolved in commit _"Update swagger to remove readOnly for PUT and POST parameters and add required properties"_